vowpalwabbit/core/src/array_parameters_dense.cc

```diff
@@ -89,6 +89,26 @@ void dense_parameters::clear_offset(const size_t offset, const size_t params_per
   }
 }
 
+void dense_parameters::adjust_weights_single_model(const size_t params_per_problem, const size_t model_num)
```

Comment:
this method could also be unit tested in `weights_test.cc` seems like a prime candidate for some rigorous unit testing

Reply:
The weights are tested here:
https://github.com/VowpalWabbit/vowpal_wabbit/pull/4279/files#diff-9932fc58f28480ef64a94f6fc4cb26b0b50e7d94c621a3edb00fd0131fcc14dfR333

It doesn't make much sense to test this without initializing weights with a standard and automl model since it converts one to the other